stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1361,18 +1358,17 @@ end
     # The largest positive floating point number whose inverse is less than infinity
     δ = inv(prevfloat(typemax(nrm)))
 
-    if nrm == Inf  # Just divide since performance isn't important in this corner case
-        v ./= Inf
-    elseif nrm ≥ δ # Safe to multiply with inverse
+    if nrm ≥ δ # Safe to multiply with inverse
         invnrm = inv(nrm)
         rmul!(v, invnrm)
-    else           # Scale elements to avoid overflow
+
+    else # scale elements to avoid overflow
         εδ = eps(one(nrm))/δ
         rmul!(v, εδ)
         rmul!(v, inv(nrm*εδ))
     end
 
-    return v
+    v
```

Comment:
-1 for removing explicit returns.

Reply:
I'm ok for both style. But I want the consistency of the same module.

Reply:
I added the `return` in the commit I'm reverting and I'm no longer touching that function. I also prefer explicit return statements.